tasks/run-local.sh

```diff
@@ -3,10 +3,36 @@
 # You can run against a tasks container tag different than latest by setting "$TASKS_TAG"
 set -eu
 
+DAEMON=
+TASK_SECRETS=
+
+while getopts "dhs:" opt; do
+    case $opt in
+        h)
+            echo -d runs the cockpit-tasks container as daemon
```

Comment:
Fine for me to make that the default mode, it's probably better for local development as well - then you can work with the thing without it getting ripped away under your feet after a minute.

Reply:
oh it still gets ripped away, as the tasks container exits after 30 `run-queues`